src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -66,9 +67,11 @@ export const App = ({ displayToggle, toggleLoginModal }) => {
           }
         })
         .catch(() => {
+          recordEvent({ event: '1095b-available-forms-system-error' });
```

Comment:
I debated whether this should be called `1095b-system-error` but there don't appear to be any other paths to system errors so 🤷

Reply:
I think I like the event name you chose, because in the future the app may have other possible system errors we'd want a separate name for.